pkg/catalog/catalog.go

```diff
@@ -466,16 +466,6 @@ func isSupportedImage(imgName string) bool {
 		"centos/nodejs-12-centos7:latest",
 		"rhscl/nodejs-10-rhel7:latest",
 		"rhscl/nodejs-12-rhel7:latest",
-		"bucharestgold/centos7-s2i-nodejs:latest",
-		"nodeshift/centos7-s2i-nodejs:latest",
-
-		// older images which we should remove soon
-		"rhoar-nodejs/nodejs-8:latest",
-		"rhoar-nodejs/nodejs-10:latest",
```

Comment:
this seems like a nodejs-10 image, should we remove this too?

Reply:
Unfortunately i have removed it. Will add it back